src/content/playwright/configure.mdx

```diff
@@ -68,6 +68,7 @@ These options control how the Chromatic archive fixture behaves.
 | Option                   | Type            | Description                                                                                                                                                                          |
 | ------------------------ | --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | `disableAutoSnapshot`    | `boolean`       | When `true`, it will disable the snapshot that happens automatically at the end of a test when using the Chromatic test fixture.                                                     |
+| `cropToViewport     `    | `boolean`       | When `true`, the snapshot will be clipped to viewport.height.                                                                                                                        |
```

Comment:
@cgbl-90 just a minor thing, but when you can, can you adjust the text as follows:
```suggestion
| `cropToViewport     `    | `boolean`       | When `true`, the snapshot will be clipped to the height of the [viewport](/docs/viewports#playwright).                                                                                                                        |
```